CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @hmrc/platform-security
```

Comment:
What about adding us only if they change something in the code

```
/src/main/scala/uk/gov/hmrc/secure/  @hmrc/platform-security
```

I think that we do not need to review tests, `.gitignore`, etc.

Reply:
In this case we prefer a wider view, even if that means we get some less relevant review requests.